Project-Hotpot/Project-Hotpot/Parse-Backend/PFPlaylist.swift

```diff
@@ -38,12 +38,12 @@ class PFPlaylist: PFObject, PFSubclassing {
     class func addSongtoPlaylistInBackground(song: PFSong, playlist: PFPlaylist, completion: @escaping ((Bool, Error?)-> Void)){
         // create an entry in the Follow table
         let joinTable = PFObject(className: "SongJoinTable")
-        joinTable.setObject(song, forKey: "Song")
-        joinTable.setObject(playlist, forKey: "Playlist")
+        joinTable.setObject(song, forKey: "song")
+        joinTable.setObject(playlist, forKey: "playlist")
         
         let currentDate = NSDate()
-        joinTable.setObject(currentDate, forKey: "AddedAt")
-        joinTable.setObject(currentDate, forKey: "LastPlayed")
+        joinTable.setObject(currentDate, forKey: "addedAt")
```

Comment:
If these keys repeat in multiple classes, best practice would be to create a Constant Class and put these there and use the constants everywhere. Avoids any human error